README.md

```diff
@@ -1,6 +1,8 @@
 
 # :camping: Welcome to the *Intro To Serverless* Cabin!
 
+![github](https://user-images.githubusercontent.com/28051494/114525441-f3d6c500-9bfa-11eb-9de1-421cf1b2f263.png)
```

Comment:
I'm considering placing the banner above the "Welcome" title - @lazyplatypus thoughts?